app/backend/src/couchers/tasks.py

```diff
@@ -140,6 +140,20 @@ def send_friend_request_email(friend_relationship):
         },
     )
 
+def send_friend_request_accepted_email(friend_relationship):
+    friend_requests_link = urls
```

Comment:
This should be a link to the person's profile who received the original friend req.

It should be `urls.profile_link(friend_relationship.to_user.username)` or something. You might need to add a `profile_link` function into `urls.py`. The right URL is in the frontend somewhere (see the comment at the top of `urls.py`)